python/unblob/handlers/executable/elf.py

```diff
@@ -31,6 +36,48 @@
 
 @attrs.define(repr=False)
 class ElfChunk(ValidChunk):
+    def upx_get_loader(self, file: File) -> int:
+        file.seek(-4, io.SEEK_END)  # last 4 bytes indicates where linfo ends
+        return convert_int32(file.read(4), Endian.LITTLE) - 12
+
+    def upx_calc_checksum(self, file: File, l_info: int, elf) -> bool:
+        file.seek(l_info, io.SEEK_SET)
+        l_checksum = convert_int32(file.read(4), Endian.LITTLE)
+        file.seek(l_info + 8, io.SEEK_SET)
+        l_lsize = convert_int16(file.read(2), Endian.LITTLE)
+        size_pack2 = elf.last_offset_segment - l_lsize
+        size_aligment = size_pack2  # Calc size before alignment
+        while size_aligment % 4 != 0:  # Forces to be mod 4 because of alignment
+            size_aligment += 1
+        xct_off = False  # For shared android library -> .so
+        for section in elf.sections:
+            if section.name == ".init":
+                xct_off = True
+                break
```

Comment:
I don't understand the comment "For shared android library -> .so", what does that mean ?

Can be simplified to:

```suggestion
        xct_off = any(section.name == "init" for section in elf.sections)
```

Reply:
Marked as resolved but still there.